app/models/criterion.rb

```diff
@@ -37,28 +37,37 @@ def self.assign_all_tas(criterion_ids, ta_ids, assignment)
   def self.assign_tas(criterion_ids, ta_ids, assignment)
```

Comment:
It's the same comment as the first one. Need `criterion_ids` to now have both ids and type.